README.md

```diff
@@ -1 +1,45 @@
-# moneymaker
\ No newline at end of file
+# moneymaker
+
+budget calculator
+1. user input (hardcoded)
+    1. String date
+    1. String description
+    1. int amount
+    1. int outgoing
+1. display to user
+    1. int balance projection
+
+Classes in use:
+1. Moneymaker (main method)
+    - creates a new Budget using:
+    - new Budget(new Balance(), new TransactionMaker().getTransactions()).printReport();
+    Todo: is TransactionMaker really the most simple way to get data into the system?
+1. Budget:
+    - Budget(Balance balance, List<Transaction> transactions)
+    - updateBalance()
+    - printBalance()
+1. Balance:
+    - get()
+    - increment(int i)
+1. TransactionMaker
+    - Hardcode the transactions with makeTransaction()
+    - Uses the Transaction object
+1. Transaction: (a POJO)
+     - String time
+     - TransactionType transactionType: (enum CREDIT/DEBIT)
+     - int amount
+     - String description
+     - String reportStringFormat()
+     Todo: should reportStringFormat() be in it's own class?
```

Comment:
I see where you're coming from here. Some people would put this in another class because they want this class to be as POJO-y as possible. I'm not 100% convinced of the merit of extracting it here.

There's a way of making this a bit more pure. We can override the `toString()` method instead of using this name. All objects have this method anyway and the default implementation doesn't give you much value.

I'll leave the decision up to you:

1. Leave as is
1. Extract a separate class for formatting a transaction as a string.
1. Override `toString()`